source/data-formats/dates-and-times.txt

```diff
@@ -0,0 +1,448 @@
+.. _pymongo-dates-times: 
+
+Dates and Times
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: convert, span, central, mountain, pacific, eastern, calendar 
+
+Overview
+--------
+
+In this guide, you can learn how to handle Python ``datetime`` objects
+correctly in {+driver-short+}.
+
+.. _pymongo-datetime-terminology:
+
+Terminology
+-----------
```

Comment:
(Debated with myself on this one) Should this be a heading-level under Overview? The rest of the headings are action-based which makes me thing yes, but this also seems integral to understanding the rest of the page so keeping it as a main header might make it less likely to be skipped over.

I think I'm okay going either way, but wanted to bring it up just in case you may have a stronger opinion